Sources/PointFree/SiteMiddleware.swift

```diff
@@ -116,6 +116,10 @@ private func render(conn: Conn<StatusLineOpen, T3<Database.Subscription?, Databa
       return conn.map(const(user .*. subscriberState .*. route .*. subRoute .*. unit))
         |> blogMiddleware
 
+    case .discounts:
```

Comment:
not currently doing anything with the organization name in the route. we could add a referrals column to subscriptions tho. whatdya think?

Reply:
Down if it'd be useful! Can still get the info from Stripe, though? As long as we somehow link the coupons?

Reply:
yeah all the info should be in stripe. can either back port it or see if there's a way to generate reports with that info